apps/gitness/src/pages/pipeline-edit/theme/monaco-theme.ts

```diff
@@ -9,38 +9,93 @@ const harnessLightTheme = {
   colors: {}
 }
 
-const getHarnessDarkTheme = (options?: { isBlame?: boolean }) => ({
+const getHarnessDarkTheme = () => ({
```

Comment:
Does this need to be in a function anymore?

Reply:
We can proceed with same colors for blame and editor. This was there only because of color for line numbers that was different in initial mocks. But this look great now.

<img width="1206" alt="Screenshot 2024-12-12 at 14 19 30" src="https://github.com/user-attachments/assets/2cd4aef8-4862-4e8b-b584-e0f622d09c1f" />

Reply:
fixed, thanks!